### DISCUSSION TITLE
Graduation 2023

### DISCUSSION THREAD

**Discussion by KaranBanjade** [community/community#56632] [GitHub Education]
### Hi everyone,

I have seen the repositories for Github Graduation 2019-22. Anyone have any idea about 2023 registrations or did I miss it already?

**Comment by valleyOoO** [comment]
I was looking for this year's (2023) too! Can't seem to find it though.

**Reply by LamThanhNgan** [reply]
me too 😃

**Comment by Maanayush10** [comment]
I also haven't heard it till now, but I have seen other parallel graduations happening, 
like this one:
https://github.com/AllInOpenSource/All-In-For-Students-Graduation-2023

**Comment by zyn10** [comment]
Comming Soon

**Comment by harshchhabra37** [comment]
Unfortunately, there have been no updates so far.

**Comment by Danuragtiwari** [comment]
When will the github graduation 2023 is going to start ,is their any update?

**Reply by mr-desilva** [reply]
🤔 no updates yet

**Reply by pranitrathod** [reply]
I guess its over @Danuragtiwari @mr-desilva if you find anything please write here once!

**Reply by LamThanhNgan** [reply]
no updates? 🥲

**Reply by pranitrathod** [reply]
@LamThanhNgan you too 2023 grad?
